src/global.ts

```diff
@@ -117,6 +117,9 @@ interface RedirectLoginResult {
 }
 
 interface PopupLoginOptions extends BaseLoginOptions {}
+interface PopupConfigOptions {
```

Comment:
docs please, anywhere? Maybe on the README?

Reply:
done